packages/@react-types/numberfield/src/index.d.ts

```diff
@@ -23,15 +23,21 @@ import {
 } from '@react-types/shared';
 
 export interface NumberFieldProps extends InputBase, Validation, FocusableProps, TextInputBase, ValueBase<number>, RangeInputBase<number>, LabelableProps {
+  /** Accessible label for the decrement button. */
   decrementAriaLabel?: string,
+  /** Accessible label for the increment button. */
   incrementAriaLabel?: string,
+  /** The display format of the value. */
   formatOptions?: Intl.NumberFormatOptions
 }
 
 export interface AriaNumberFieldProps extends NumberFieldProps, DOMProps, AriaLabelingProps {}
 
 export interface SpectrumNumberFieldProps extends AriaNumberFieldProps, StyleProps, SpectrumLabelableProps, Omit<AriaNumberFieldProps, 'placeholder'> {
```

Comment:
The omit doesn't seem to be working. I think because `AriaNumberFieldProps` is there twice 🤦